ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -212,6 +212,14 @@ func ResourceIBMContainerVpcWorkerPool() *schema.Resource {
 				Computed:    true,
 				Description: "Autoscaling is enabled on the workerpool",
 			},
+
+			"security_groups": {
```

Comment:
can you please add the ApplyOnce diff suppress here too?

Reply:
sure, will do